code/game/objects/items/weapons/twohanded.dm

```diff
@@ -287,7 +287,9 @@
 	if(W.is_screwdriver(user) && user.is_holding_item(src))
 		playsound(src.loc, 'sound/items/Screwdriver.ogg', 50, 1)
 		to_chat(user, "<span class='notice'>You detach [src] from your arm.</span>")
-		user.drop_item(src, force_drop=1)
+		new /obj/item/weapon/melee/energy/hfmachete(get_turf(user.loc))
```

Comment:
Man, just use `usr.loc`

Reply:
Has the peculiar result of making someone who died in a locker drop their weapons out of the locker instead of into it